backend/src/main/resources/application-dev.yml

```diff
@@ -17,12 +17,14 @@ security:
     client:
       client-id: xbdd
       client-secret: secret
+      accessTokenUri: http://localhost:8080/oauth/token
 
-server:
-  port: 8443
-  ssl:
-    key-store: classpath:certs/keystore.p12
-    key-store-password: password
+# Enable this if you want to run on https, note that the frontend .env files will need to be updated
```

Comment:
Do we actually want to turn off ssl now that we're forcing actual authentication? Isn't that unsafe? Unless we have some kind of dev-only login option (like a mock LDAP server or something like that)?

Reply:
I think for dev/ng its fine but in a deployed scenario I think we would definitely want it. Its why I left all the instructions in there. If you get a hold of a cert its trivial to turn on

Reply:
While I think of it we should start building up a how to deploy readme file at some stage as the old one is still focussed around the old xbdd. It would be good to get ldap integration happening then I think its basically good to replace it.

Reply:
We did talk about reverting to http, but I've since had a play around with it and I'm just trying to get my head around how this works. I suppose the question is - if we were to do xbdd dev on a public network, would someone be able to steal our github token?

Reply:
No because the token is retrieved on the backend then thrown away. They could get out xbdd token though, all that will give them access to is the xbdd backend. This is another reason why you should throw the github token away as soon as you have authenticated the user.